vrc-get-gui/locales/en.json5

```diff
@@ -295,6 +299,8 @@
 
     "vpm repositories:dialog:downloading...": "Downloading the repository...",
     "vpm repositories:dialog:already added": "This repository was already added.",
+    "vpm repositories:dialog:id duplicated": "The repository ID is same as repository named {{name}}.",
+    "vpm repositories:dialog:url duplicated": "The repository ID is same as repository named {{name}}.",
```

Comment:
> The repository ID is

this message is for url duplicated, so maybe it should not be "ID"

Reply:
yes. my mistake

Reply:
also, repository name should be written with `<b>`, maybe